common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -24,3 +24,45 @@ fun errorLog(error: PurchasesError) {
         Log.e("[Purchases] - ERROR", error.toString())
     }
 }
+
+fun debugLog(emojiList: List<String>, message: String) {
+    if (Config.debugLogsEnabled) {
+        var len = 1
+        for (emoji in emojiList) {
+            len += emoji.length
+        }
+        val emojifiedMessage = emojiList.joinToString("").padEnd(len).plus(message)
```

Comment:
What's this padding for?

Reply:
It's to add an empty space between the emoji list and the message. Mainly to prevent starting the const in the string file as a space. 
E.g: "😻 Message starts here" instead of "😻Message starts here"

Reply:
would string formatting perhaps help? 
i.e.: 
```kotlin
val emojifiedMessage = "${emojiList.joinToString("")} $message"
```